README_fr-FR.md

```diff
@@ -0,0 +1,443 @@
+ Date: Sun Feb 25 23:14:23 2018 -0300
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+```
+
+### add/checkout/reset [--patch | -p]
+
+Immaginons que nous avons le diff suivant:
+
+```diff
+diff --git a/README.md b/README.md
+index 7b45277..6b1993c 100644
+--- a/README.md
++++ b/README.md
+@@ -186,10 +186,13 @@ bebebe Corrige nome de método na classe InventoryBackend
+ ``
+ # Bad (mixes English and Portuguese)
+ ababab Usa o InventoryBackendPool para recuperar o backend de estoque
+-efefef Add `use` method to Credit model
+ cdcdcd Agora vai
+ ``
+
++### Modèle
++
++C'est un modèle, [Ecrit par Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), qui apparait dans [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
++
+ ## Contribuer
+
+ Toute sorte d'aide est apréciée. Voici quelques exemples de sujets où vous pouvez m'aider. 
+
+@@ -202,3 +205,4 @@ Toute sorte d'aide est apréciée. Voici quelques exemples de sujets où vous pouvez m'aider. 
+
+ - [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
+ - [Pro Git Book - Commit guidelines](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines)
++- [A Note About Git Commit Messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+```
+
+Ont peut utiliser 'git add -p' pour ajouter seulement les patch que nous souhaitons, sans le besoin de changer le code qui est déjà écrit.
+C'est utile de séparer un gros changement en plusieurs commits ou de reset/checkout certains changements spécifiques.
+
+```
+Stage this hunk [y,n,q,a,d,/,j,J,g,s,e,?]? s
+Split into 2 hunks.
+```
+
+#### hunk 1
+
+```diff
+@@ -186,7 +186,6 @@
+ ``
+ # Bad (mixes English and Portuguese)
+ ababab Usa o InventoryBackendPool para recuperar o backend de estoque
+-efefef Add `use` method to Credit model
+ cdcdcd Agora vai
+ ``
+
+Stage this hunk [y,n,q,a,d,/,j,J,g,e,?]?
+```
+
+#### hunk 2
+
+```diff
+@@ -190,6 +189,10 @@
+ ``
+ cdcdcd Agora vai
+ ``
+
++### Template
++
++This is a template, [written originally by Tim Pope](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html), which appears in the [_Pro Git Book_](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project).
++
+ ## Contributing
+
+ Any kind of help would be appreciated. Example of topics that you can help me with:
+Stage this hunk [y,n,q,a,d,/,K,j,J,g,e,?]?
+
+```
+
+#### hunk 3
+
+```diff
+@@ -202,3 +205,4 @@ Any kind of help would be appreciated. Example of topics that you can help me wi
+
+ - [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
+ - [Pro Git Book - Commit guidelines](https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines)
++- [A Note About Git Commit Messages](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html)
+```
+
+## D'autres choses intéréssantes
+
+https://whatthecommit.com/
```

Comment:
A link to https://gitmoji.carloscuesta.me/ is missing. See d7aaa9de7cb62bc542a49230d17c5fc943c1297d for reference.

```suggestion
- https://whatthecommit.com/
- https://gitmoji.carloscuesta.me/
```